### DISCUSSION TITLE
Errore totale generazione ricevuta

### DISCUSSION THREAD

**Discussion by KromoDev** [fattureincloud/api#347] [Q&A]
Ciao a tutti! Sto generando delle ricevute, non riesco a capire come mai inviando questo array:

`array(2) {
  ["data"]=>
  array(8) {
    ["type"]=>
    string(7) "receipt"
    ["entity"]=>
    array(1) {
      ["name"]=>
      string(16) "Visitatore parco"
    }
    ["payments_list"]=>
    array(1) {
      [0]=>
      array(5) {
        ["due_date"]=>
        string(10) "2023-08-13"
        ["amount"]=>
        float(24.590163934426)
        ["payment_account"]=>
        array(1) {
          ["id"]=>
          int(458248)
        }
        ["status"]=>
        string(4) "paid"
        ["paid_date"]=>
        string(10) "2023-08-13"
      }
    }
    ["items_list"]=>
    array(1) {
      [0]=>
      array(4) {
        ["name"]=>
        string(17) "Biglietti on-line"
        ["qty"]=>
        int(1)
        ["net_price"]=>
        float(24.590163934426)
        ["vat"]=>
        array(2) {
          ["id"]=>
          int(0)
          ["value"]=>
          int(22)
        }
      }
    }
    ["amount_total"]=>
    float(24.590163934426)
    ["rc_center"]=>
    string(23) "Acropark Laghetto Roana"
    ["e_invoice"]=>
    bool(true)
    ["ei_data"]=>
    array(1) {
      ["payment_method"]=>
      string(4) "MP08"
    }
  }
  ["options"]=>
  array(1) {
    ["fix_payments"]=>
    bool(true)
  }
}`

venga generata una ricevuta con totale 586.93, più nello specifico: 

`"data": [
    {
      "id": 299993223,
      "type": "receipt",
      "numeration": "",
      "subject": "",
      "visible_subject": "",
      "amount_net": 481.09,
      "amount_vat": 105.84,
      "amount_gross": 586.93,
      "amount_due_discount": 0,
      "entity": {
        "name": "Visitatore parco",
        "vat_number": "",
        "tax_code": "",
        "address_street": "",
        "address_postal_code": "",
        "address_city": "",
        "address_province": "",
        "address_extra": "",
        "country": "Italia",
        "certified_email": "",
        "ei_code": "",
        "entity_type": "client",
        "type": null
      },
      "date": "2023-08-14",
      "number": 456,
      "next_due_date": null,
      "url": "https://compute.fattureincloud.it/doc/lwry4covxislxk10nvxpzeuz399imsjk.pdf"
    },`
    
    Grazie in anticipo a chi sarà così gentile da aiutarmi

**Comment by emajo** [comment]
Ciao @KromoDev, le chiedo gentilmente se può inviare il json completo del documento prima dell'invio (json_encode($array)), eventualmente censurando i dati sensibili, per capire meglio dove sta il problema.

Grazie,
Emanuele.

**Reply by valmoz** [reply]
Buongiorno @KromoDev è riuscito a risolvere il problema?

Grazie
Mauro
